modules/roomify/roomify_travel_theme/roomify_travel_theme.pages_default.inc

```diff
@@ -2039,10 +2039,10 @@ function roomify_travel_theme_default_page_manager_handlers() {
     ),
   );
   $display->cache = array();
-  $display->title = '';
+  $display->title = '%node:title_field';
   $display->uuid = '9ef84618-1857-465c-a31a-7230c09fde85';
   $display->storage_type = 'page_manager';
-  $display->storage_id = 'node_view__location';
+  $display->storage_id = 'node_view__location-travel';
```

Comment:
@salvoscala is this anything to do with the travel theme? do we also need something for roomify bootstrap?